share/lib/reports.awk

```diff
@@ -0,0 +1,179 @@
+  if ($1 != "R: ")
+  {
+    if (multiline && info)
+    {
+      print darkgreen $0 normal;
+      next
+    } else {
+      # not a Rudder report
+      next
+    }
+  }
+  # Wait for the StartRun to start display because we want the config id
+  if ($8 == "StartRun")
+  {
+    printf "%s\nNode uuid: %s\n%s\n", version, uuid, $11;
+    if (!multiline)
+    {
+      printf "\n%s%-8.8s %-25.25s %-25.25s %-18.18s %s%s\n", white, "Result", "Technique", "Component", "Key", "Message", normal;
+    }
+    next
+  }
+  if ($8 == "EndRun")
+  {
+    # skip this one
+    next
+  }
+  
+  if ($3 == "result_success")
+  { 
+    color = green;
+    result = "success";
+    success++;
+  }
+  else if ($3 == "result_error")
+  {
+    color = red;
+    result = "error";
+    error++;
+  }
+  else if ($3 == "result_na")
+  {
+    color = green;
+    result = "n/a";
+  }
+  else if ($3 == "result_repaired")
+  {
+    color = yellow;
+    result = "repaired";
+    repaired++;
+  }
+  else if ($3 == "log_info" || $3 == "log_repaired" || $3 == "log_debug")
+  {
+    if (!info)
+    {
+      next
+    }
+    color = cyan;
+    result = "info";
+  }
+  else
+  {
+    color = white;
+    result = $3;
+  }
+  if ($8 == "None")
+  { 
+    # Do not diplay "None" keys
+    $8 = "";
+  }
+  
+  {
+    if (multiline)
+    {
+      if (current_technique != $2)
+      {
+        current_technique = $2;
+        printf "\n%s%-80.80s%s\n", white, "## Technique: " $2 " " padding, normal;
+        new_technique = 1;
+      } else {
+        new_technique = 0;
+      }
+
+      if (new_technique || current_component != $7)
+      {
+        current_component = $7;
+        printf "%s%-80.80s%s\n", white, "-- Component: " $7 " " padding_dash, normal;
+      }
+
+      if ($8 != "")
+      {
+        printf "%s** Key: %s%s\n", white, $8, normal;
+      }
+
+      printf "%s%s%s: %s\n", color, result, normal, $11;
+    }
+    else {
+      printf "%s%-8.8s%s ", color, result, normal;
+
+      if (length($2) > 25)
+      {
+        printf "%-24.24s| ", $2;
+      } else {
+        printf "%-25.25s ", $2;
+      }
+
+      if (length($7) > 25)
+      {
+        printf "%-24.24s| ", $7;
+      } else {
+        printf "%-25.25s ", $7;
+      }
+
+      if (length($8) > 18)
+      {
+        printf "%-17.17s| ", $8;
+      } else {
+        printf "%-18.18s ", $8;
+      }
+
+      printf "%s\n", $11
+
+    }
+    fflush();
+  }
+}
+END {
+  "date +%s.%N" | getline endtime;
+
+  if (multiline)
+  {
+    printf "\n%s%-80.80s%s\n", white, "## Summary " padding, normal
+  }
+  else
+  {
+    printf "\n%sSummary%s\n", white, normal
+  }
+
+  if (success > 0)
+  {
+    printf "success: %s%5.5s%s\n", green, success, normal
+  }
+  if (repaired > 0)
+  {
+    printf "repaired: %s%4.4s%s\n", yellow, repaired, normal
+  }
+  if (error > 0)
+  {
+    printf "error: %s%7.7s%s\n", red, error, normal;
+  }
+
+  printf "compliance: %.2f%\n", 100 * (success + repaired)  / (success + repaired + error);
```

Comment:
I think you should not display it at all
Difference of computation within the webapp is already sufficiently confusing, do not add anymore source of confusion.